test/unit/permission/authorization/PermissionMap.test.js

```diff
@@ -127,7 +127,7 @@ describe('permission', function () {
           // THEN
           expect(redirectStateName).toBePromise();
           expect(redirectStateName).toBeResolvedWith({state: 'adminRedirect'});
-          expect(adminRedirectTo).toHaveBeenCalledWith('ADMIN', jasmine.any(Object));
+          expect(adminRedirectTo).toHaveBeenCalledWith(PermRoleStore, 'ADMIN');
```

Comment:
These are used to assert the passed in, arbitrary injection values declared on the above $inject = [] assignment